facts/Vagrantfile

```diff
@@ -64,6 +64,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     host.vm.provision 'shell', path: 'get_facts.sh'
     host.vm.provision 'shell', inline: '/sbin/shutdown -h now'
   end
+  config.vm.define 'ubuntu-22.04-arm64', autostart: false do |host|
+    host.vm.box = 'bento/ubuntu-22.04'
```

Comment:
how does it work? Vagrant checks that you're running ARM64 locally and then it automatically downloads the ARAM64 version of that box?